src/game/server/entities/character.cpp

```diff
@@ -1059,7 +1060,7 @@ void CCharacter::FireWeapon()
 
 		case WEAPON_SHOTGUN:
 		{
-			int ShotSpread = 3;
+			int ShotSpread = 5;
```

Comment:
Will this increase the spread of all shotguns? Not saying that it is bad, maybe it is a good thing, cant tell without testing.

Reply:
I wonder why I touched that variable.
Yes it will change the behaviour of all humans and increases bullet number.
I will revert that change.